src/index.html

```diff
@@ -1,7 +1,7 @@
 ---
 layout: homepage
 title: Dart programming language
-description: Dart is an application programming language that’s easy to learn, easy to scale, and deployable everywhere.
+description: Dart is a language uniquely optimized for client-side development for web and mobile.
```

Comment:
"uniquely" seems unnecessary and sounds like bragging to me. But if we really want to say this and will back it up, I'm OK with it. (This description shows up in search engine results and social shares.)

Reply:
Happy to leave out uniquely for now. @kevmoo also reacted to the use of this word in a different context.